x/gamm/keeper/export_test.go

```diff
@@ -6,16 +6,15 @@ import (
 	"github.com/osmosis-labs/osmosis/v12/x/gamm/types"
 )
 
-// SetParams sets the total set of params.
-func (k Keeper) SetParams(ctx sdk.Context, params types.Params) {
-	k.setParams(ctx, params)
-}
-
 // SetPool adds an existing pool to the keeper store.
 func (k Keeper) SetPool(ctx sdk.Context, pool types.PoolI) error {
 	return k.setPool(ctx, pool)
 }
 
-func (k Keeper) GetNextPoolIdAndIncrement(ctx sdk.Context) uint64 {
-	return k.getNextPoolIdAndIncrement(ctx)
+// SetNumPools sets the number of pools to a given number.
+func (k Keeper) SetPoolCount(ctx sdk.Context, poolCount uint64) {
+	k.initializePoolCount(ctx)
+	for i := uint64(0); i < poolCount; i++ {
+		k.incrementPoolCount(ctx)
+	}
```

Comment:
I think this isn't an efficient way of storing given that we already have the poolCount that we have to set to(since we're rewriting and storing every iteration). I think we should create a new method called`setPoolCount` that could set the value directly

Reply:
This was removed, not even sure what this is